nucleus/core/logging/src/main/java/com/sun/enterprise/server/logging/logviewer/backend/LogFilterForInstance.java

```diff
@@ -98,7 +100,7 @@ public File downloadGivenInstanceLogFile(ServiceLocator habitat, Server targetSe
             String loggingDir = getLoggingDirectoryForNode(instanceLogFileName, node, sNode, instanceName);
 
             try {
-                Vector instanceLogFileNames = sftpClient.ls(loggingDir);
+                List instanceLogFileNames = sftpClient.ls(loggingDir);
```

Comment:
Can probably add the right generic param here

Reply:
Cheers, good spot! Made that change now